packages/swapper/src/utils.ts

```diff
@@ -169,16 +170,25 @@ export const checkEvmSwapStatus = async ({
   txHash,
   chainId,
   assertGetEvmChainAdapter,
+  getSafeApiKit,
 }: {
   txHash: string
   chainId: ChainId
   assertGetEvmChainAdapter: (chainId: ChainId) => EvmChainAdapter
+  getSafeApiKit: (chainId: EvmChainId, safeAddress: string) => Promise<SafeApiKit>
 }): Promise<{
   status: TxStatus
   buyTxHash: string | undefined
   message: string | undefined
 }> => {
   try {
+    const safeApiKit = await getSafeApiKit(
+      chainId as EvmChainId,
+      '0x00342Dc97c6419089130bB2F1a39c1Bfea4fFe40',
```

Comment:
Note, this is hardcoded to the testing safe I've used, sell address should obviously be programmatic